src/script/properties/PropertiesRepository.js

```diff
@@ -93,7 +93,7 @@ class PropertiesRepository {
 
   /**
    * Get the current preference for a property type.
-   * @param {PROPERTIES_TYPE} propertiesType - Type of preference to get
```

Comment:
What was wrong about `PROPERTIES_TYPE`?

Reply:
what is passed there is actually a string, `PROPERTIES_TYPE` is an object. (remember, we discovered this together :) )

Reply:
Whoops, true!